src/System.Security.Cryptography.Cng/ref/project.json

```diff
@@ -3,13 +3,12 @@
     "System.Runtime": "4.1.0",
     "System.IO": "4.1.0",
     "System.Runtime.Handles": "4.0.1",
-    "System.Security.Cryptography.Algorithms": "4.2.0",
     "System.Security.Cryptography.Primitives": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.6": {
+    "netstandard1.7": {
       "imports": [
-        "dotnet5.5"
+        "dotnet5.8"
```

Comment:
I don't know that we need a dotnet5.x import for netstandard1.7.  And if we did need one I think it would be 5.6, not 5.8